data/css/modules/card/_legacyPolaroid.scss

```diff
@@ -7,7 +7,7 @@
         padding: 4px 0px;
         color: $accent-dark-color;
         font-family: $title-font, sans-serif;
-        font-size: 0.77em;
+        font-size: 0.67em;
```

Comment:
Instead of affecting the font size for the majority of apps that work fine, I'd prefer changing this font size only for the four affected apps that use the Bangladesh font, in their overrides.scss in eos-subscriptions-apps.